### ISSUE TITLE
Some odd names to check

### ISSUE THREAD

**Issue by ManonGros** [plazi/community#304]
Hi PLAZI,

Some of the names reported in [this issue](https://github.com/gbif/backbone-feedback/issues/21) come from data published by PLAZI. They all seem to be oddly shaped. Would it be possible to check them out?

- [x] [Mus cypriacus al, 2006](https://www.gbif.org/species/11784369) - Source: https://www.gbif.org/dataset/a016af63-6437-427b-80b7-22bc9a002e20; [E2099A0D3426FF97E1372C0977498313](https://treatment.plazi.org/GgServer/summary/E2099A0D3426FF97E1372C0977498313)
- [x] [Pepsis mexicana hawk, 1895](https://www.gbif.org/species/10850699) - Source: https://www.gbif.org/dataset/8cd88652-1627-4fbe-891d-062b3e02e511; [91350F1CFFB3FF80073DFFE4FFADFFC4](https://treatment.plazi.org/GgServer/summary/91350F1CFFB3FF80073DFFE4FFADFFC4)
- [ ] [Mecopelidnota bondili litt](https://www.gbif.org/species/11112793) - Source: https://www.gbif.org/dataset/26433639-8de9-4195-8d16-19ce8bcffd7e
- [ ] [Mecopelidnota willersi litt](https://www.gbif.org/species/11206408) - Source: https://www.gbif.org/dataset/26433639-8de9-4195-8d16-19ce8bcffd7e
- [ ] [Pelidnota auripes litt](https://www.gbif.org/species/11164852) - Source: https://www.gbif.org/dataset/26433639-8de9-4195-8d16-19ce8bcffd7e
- [ ] [Pelidnota demergesi litt](https://www.gbif.org/species/11192307) - Source: https://www.gbif.org/dataset/26433639-8de9-4195-8d16-19ce8bcffd7e
- [ ] [Pelidnota desantacatarina litt](https://www.gbif.org/species/11160059) - Source: https://www.gbif.org/dataset/26433639-8de9-4195-8d16-19ce8bcffd7e
- [ ] [Pelidnota rectificata litt](https://www.gbif.org/species/11132748) - Source: https://www.gbif.org/dataset/26433639-8de9-4195-8d16-19ce8bcffd7e
- [ ] [Sorocha ferrandi litt](https://www.gbif.org/species/11120413) - Source: https://www.gbif.org/dataset/26433639-8de9-4195-8d16-19ce8bcffd7e
- [ ] [Astelia microsperma parte](https://www.gbif.org/species/9650910) - Source: https://www.gbif.org/dataset/b7a253a1-af73-4a27-9a60-b076d9269819
- [x] [Euphlyctina phaeopasta record](https://www.gbif.org/species/11431370) - Source: https://www.gbif.org/dataset/aea11b3e-54f2-4516-bb67-271b2f006054; [4B614497F0C85B338119C274C43AD120](https://treatment.plazi.org/GgServer/summary/4B614497F0C85B338119C274C43AD120)
- [x] [Smaragdina divisa record](https://www.gbif.org/species/11504642) - Source: https://www.gbif.org/dataset/ba2572cb-ad18-4a84-97ab-268c8332e4fd; [C207BF1CD6BB50DB836ED0D0FED79BD9](https://tb.plazi.org/GgServer/summary/C207BF1CD6BB50DB836ED0D0FED79BD9)
- [x] [Smaragdina laboissierei record](https://www.gbif.org/species/11410426) - Source: https://www.gbif.org/dataset/ba2572cb-ad18-4a84-97ab-268c8332e4fd; [C207BF1CD6BB50DB836ED0D0FED79BD9](https://tb.plazi.org/GgServer/summary/C207BF1CD6BB50DB836ED0D0FED79BD9); fixed name https://www.gbif.org/species/192416226
- [x] [Pericalus formosanus resurr](https://www.gbif.org/species/11616053) - Source: https://www.gbif.org/dataset/31bce27d-7094-467b-95ae-009a27e00cf5; [AA26FF84157BFFAE490E7F133744FF98](https://treatment.plazi.org/GgServer/summary/AA26FF84157BFFAE490E7F133744FF98); fixed name https://www.gbif.org/species/154125996

Let me know if you have any question, thanks!

**Comment by Carol-Sokolowicz** [comment]
I've fixed the names but don't know why it hasn't been updated on GBIF

**Comment by myrmoteras** [comment]
@Carol-Sokolowicz you can get an idea why, if the ingestion here https://www.gbif.org/system-health is high

**Comment by flsimoes** [comment]
@myrmoteras @ManonGros Some of the names reported here are invalid, are there guidelines on how to deal with them in GBIF?

Also, one of the reported names is a _pro parte_, meaning it includes more than one currently recognized entity, and that only one of those entities is being considered. Not sure how that should be presented on TB and GBIF

**Comment by flsimoes** [comment]
> I've fixed the names but don't know why it hasn't been updated on GBIF

It's been updated, but they are not fully fixed yet

**Comment by flsimoes** [comment]
Fixed the _Smaragdina_ in [C207BF1CD6BB50DB836ED0D0FED79BD9](https://tb.plazi.org/GgServer/summary/C207BF1CD6BB50DB836ED0D0FED79BD9)

They should be fixed in GBIF soon

**Comment by flsimoes** [comment]
As @Carol-Sokolowicz said, the _Pericalus_ in [AA26FF84157BFFAE490E7F133744FF98](https://treatment.plazi.org/GgServer/summary/AA26FF84157BFFAE490E7F133744FF98) should be fixed. I gave it another go to see if it finally triggers an update by GBIF

**Comment by flsimoes** [comment]
The _Mus cypriacus_ in [E2099A0D3426FF97E1372C0977498313 ](https://treatment.plazi.org/GgServer/summary/E2099A0D3426FF97E1372C0977498313) had a missing authority attribute.

**Comment by ManonGros** [comment]
Thanks @flsimoes !

> Some of the names reported here are invalid, are there guidelines on how to deal with them in GBIF?
> Also, one of the reported names is a pro parte, meaning it includes more than one currently recognized entity, and that only one of those entities is being considered. Not sure how that should be presented on TB and GBIF

Perhaps this is a question for @DianRHR and @camiplata

**Comment by flsimoes** [comment]
Regarding the non-disappearance of the wrong taxonomicNames from GBIF (it does disappear from the Plazi record though) is likely a GBIF issue. Perhaps these wrong names are not cleaned up at all after we update them...

**Comment by ManonGros** [comment]
Yes @flsimoes the link above are to the names that were already integrated in the GBIF backbone taxonomy so they won't disappear until the backbone gets updated. I checked the PLAZI datasets and the names were indeed removed from what is provided to GBIF. This means that the next backbone should be good

**Comment by flsimoes** [comment]
@Carol-Sokolowicz please check the status of this one
